src/main/java/org/cyclops/cyclopscore/tileentity/CyclopsTileEntity.java

```diff
@@ -51,7 +51,8 @@
     private boolean shouldSendUpdate = false;
     private int sendUpdateBackoff = 0;
     private final boolean ticking;
-    private Table<Capability<?>, EnumFacing, Object> capabilities = HashBasedTable.create();
+    private Map<Capability<?>, Object> capabilities = new HashMap<>();
```

Comment:
Replace with `Maps.newIdentityHashMap()`, I've noticed that this is a lot faster than even an ImmutableMap. (You can then also remove the copying of this unsided map in `onLoad()`)

Reply:
Also, can you rename this field to `innerCapabilities` (also its getter), for consistensy.